zstackctl/zstackctl/zstacklib.py

```
@@ -20,6 +20,9 @@
 
 from utils import log
 
+
+oemname = get_oem_name_from_properties_preload()
```

Ruff (0.14.8)

Undefined name `get_oem_name_from_properties_preload`

(F821)